modules/export_pipeline/trigger_export_pipeline.tf

```diff
@@ -0,0 +1,137 @@
+# Create bucket for predictions input.
+resource "google_storage_bucket" "predictions" {
+  name     = "${var.bucket_prefix}climateiq-predictions"
+  location = var.bucket_region
+}
+
+# Create bucket for split chunk predictions output.
+resource "google_storage_bucket" "chunk_predictions" {
+  name     = "${var.bucket_prefix}climateiq-chunk-predictions"
+  location = var.bucket_region
+}
+
+# Register a Pub/Sub topic for triggering subsequent steps in export pipeline.
```

Comment:
What is the best practice for resource ownership? This cloud function publishes messages to this Pub/Sub topic and another cloud function is triggered by these messages. Which cloud function config should own creating the topic resource? 

Similar question about buckets...one cloud function uses "climateiq-chunk-predictions" as output and another cloud function uses "climateiq-chunk-predictions" as input

Reply:
The thing to recognize is that different files in TF don't impose any degree of separation between them. If you define one resource in one file, it's still accessible in another. TF essentially concatenates all the files together when it runs, so you can define whatever file structure makes sense to you.

For instance, you could have a buckets.tf file where you put all the buckets or a shared.tf file for shared resources -- whatever makes sense to you. I'm not aware of any strong TF conventions for handling this.

Reply:
Got it, that makes sense